scripts/lint/metadata.ts

```diff
@@ -67,9 +70,23 @@ export const verifyMetadata = (
   };
 };
 
-const assertions = (repo: string) => {
+const assertions = async (repo: string) => {
   const prefix = "https://github.com/catppuccin/userstyles";
+
+  const { userstyles } = await validateYaml<UserstylesSchema>(
+    Deno.readTextFileSync(path.join(REPO_ROOT, "scripts/userstyles.yml")),
+    userStylesSchema,
+  );
+  if (userstyles === undefined) {
+    Deno.exit(1);
+  }
```

Comment:
Adapted from https://github.com/catppuccin/userstyles/blob/main/scripts/sync-maintainers/main.ts#L14-L20. Should we just make this code a function like `getUserstylesData` in `@/utils.ts`?